pkg/api/v1/validation/validation.go

```diff
@@ -60,6 +60,15 @@ func ValidateResourceRequirements(requirements *v1.ResourceRequirements, fldPath
 		allErrs = append(allErrs, validateContainerResourceName(string(resourceName), fldPath)...)
 		// Validate resource quantity.
 		allErrs = append(allErrs, ValidateResourceQuantityValue(string(resourceName), quantity, fldPath)...)
+
+		// For GPUs, limits has to be set equal to requests.
+		if resourceName == v1.ResourceNvidiaGPU {
```

Comment:
Every line of code we introduce that's per resource makes the system harder to explain.
GPUs will be kicked out to an extension sometime soon. So I don't think its worth expanding the validation logic just for GPUs.